docs/src/main/tut/oneand.md

```diff
@@ -21,10 +21,4 @@ type NonEmptyList[A] = OneAnd[List, A]
 
 which is the actual implementation of non-empty lists in cats.  By
 having the higher kinded type parameter `F[_]`, `OneAnd` is also able
-to represent other "non-empty" data structures, e.g.
```

Comment:
I don't feel strong about this but shall we replace this example with `NonEmptyStream`, instead of just removing it.